### ISSUE TITLE
Not accessible according to WCAG

### ISSUE THREAD

**Issue by suyashdeshpande** [coding-blocks/codingblocks.online.projectx#306]
The website is not accessible according to WCAG. Some of the examples are :- 
![image](https://user-images.githubusercontent.com/28438230/49495249-63e51b80-f888-11e8-8700-b0317dc80b66.png)
The bell icon and profile is not receiving focus as div is used without tabindex. A role should also be given.

![image](https://user-images.githubusercontent.com/28438230/49495316-9e4eb880-f888-11e8-8c6d-44a37e108648.png)

In this the cards that are hidden are also receiving focus but they should not. They should not be in DOM if they are not visible.
There are many more accessibility related issues.

**Comment by abhishek97** [comment]
Nice find!, would you be able make PR to improve these?

**Comment by suyashdeshpande** [comment]
Yes, I will do it
